src/fe/fe_base.C

```diff
@@ -743,38 +743,37 @@ void FEGenericBase<OutputType>::compute_shape_functions (const Elem * elem,
     this->_fe_trans->map_div(this->dim, elem, qp, (*this), this->div_phi);
 }
 
+
+// Note I am passing points instead of using members because size changes between calls
```

Comment:
This seems to be the key to the whole commit here, yet this sentence is too cryptic for me to understand it.

Reply:
Sure, let me clarify things a bit more.